### ISSUE TITLE
No module django_u2f

### ISSUE THREAD

**Issue by darkpixel** [gavinwahl/django-u2f#15]
Installs via pip:
```
17:59:15 [aaron@ender:~/code] develop* 1 ± pip install git+https://github.com/gavinwahl/django-u2f.git
Downloading/unpacking git+https://github.com/gavinwahl/django-u2f.git
  Cloning https://github.com/gavinwahl/django-u2f.git to /tmp/pip-yjKj6u-build
  Running setup.py (path:/tmp/pip-yjKj6u-build/setup.py) egg_info for package from git+https://github.com/gavinwahl/django-u2f.git

Downloading/unpacking django-argonauts (from django-u2f==0.0.0)
  Downloading django-argonauts-1.1.4.tar.gz
  Running setup.py (path:/home/aaron/.virtualenvs/code/build/django-argonauts/setup.py) egg_info for package django-argonauts

Requirement already satisfied (use --upgrade to upgrade): django>=1.8 in /home/aaron/.virtualenvs/code/lib/python2.7/site-packages (from django-u2f==0.0.0)
Downloading/unpacking qrcode (from django-u2f==0.0.0)
  Downloading qrcode-5.1.tar.gz
  Running setup.py (path:/home/aaron/.virtualenvs/code/build/qrcode/setup.py) egg_info for package qrcode

Requirement already satisfied (use --upgrade to upgrade): six in /home/aaron/.virtualenvs/code/lib/python2.7/site-packages (from django-u2f==0.0.0)
Installing collected packages: django-argonauts, qrcode, django-u2f
  Running setup.py install for django-argonauts

  Running setup.py install for qrcode

    Installing qr script to /home/aaron/.virtualenvs/bin
  Running setup.py install for django-u2f

Successfully installed django-argonauts qrcode django-u2f
Cleaning up...
18:00:26 [aaron@ender:~/code] develop(+3/-0)* 1 ± 
```

Try to run the devserver:
```
18:00:45 [aaron@ender:~/code] develop(+3/-0)* 1 ± python manage.py runserver 0.0.0.0:8000
Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/home/aaron/.virtualenvs/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 338, in execute_from_command_line
    utility.execute()
  File "/home/aaron/.virtualenvs/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 312, in execute
    django.setup()
  File "/home/aaron/.virtualenvs/local/lib/python2.7/site-packages/django/__init__.py", line 18, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/home/aaron/.virtualenvs/local/lib/python2.7/site-packages/django/apps/registry.py", line 85, in populate
    app_config = AppConfig.create(entry)
  File "/home/aaron/.virtualenvs/local/lib/python2.7/site-packages/django/apps/config.py", line 86, in create
    module = import_module(entry)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
ImportError: No module named django_u2f
18:02:25 [aaron@ender:~/code] develop(+3/-0)* 1 ±
```

Looking in site-packages directory:
```
18:04:07 [aaron@ender:~/.virtualenvs/lib/python2.7/site-packages] [] $ find | grep -i u2f
./django_u2f-0.0.0-py2.7.egg-info
./django_u2f-0.0.0-py2.7.egg-info/dependency_links.txt
./django_u2f-0.0.0-py2.7.egg-info/requires.txt
./django_u2f-0.0.0-py2.7.egg-info/SOURCES.txt
./django_u2f-0.0.0-py2.7.egg-info/PKG-INFO
./django_u2f-0.0.0-py2.7.egg-info/top_level.txt
./django_u2f-0.0.0-py2.7.egg-info/installed-files.txt
18:04:10 [aaron@ender:~/.virtualenvs/lib/python2.7/site-packages] [] $ 
```

There is no django_u2f module folder.

**Comment by gavinwahl** [comment]
I just merged #12 which changes the setup.py to fix this bug.

**Comment by darkpixel** [comment]
Thanks!
